blueocean-dashboard/src/main/resources/jenkins/plugins/blueocean/dashboard/Messages_de.properties

```diff
@@ -0,0 +1,64 @@
+Activity=Aktivit\u00E4ten
+Artifacts=Artefakte
+Author=Autor
+Branch=Ableger
+Branches=Ableger
+Build=Build
+Changes.by=\u00C4nderungen von {0}
+Changes=\u00C4nderungen
+Changes.condense={0} \u00E4nderungen
+Commit=Commit
+Completed=Komplett
+Dashboard=Instrumententafel
+Date=Datum
+Date.readable.short=MMM DD h:mma Z
+Date.readable.long=MMM DD YYYY h:mma Z
+Date.duration.format=m[ Minuten] s[ Sekunden]
+Date.duration.hint.format=M [Monate], d [Tage], h[Std.], m[m], s[s]
+Download.artifact=Herunterladen des Artefaktes
+Duration=Dauer
+Duration.param=Dauer {0}
+# the emptyState keys assume that you will parse them with a markdown parser
+EmptyState.activity=# Auf die Pl\u00E4tze, fertig...\nHmm, es liegen keine Ausf\u00fchrungen f\u00FCr vor.\n\nF\u00FCgen sie ein Commit zum Repository _{0}_ oder starten Sie die Ausf\u00fchrung per Hand.
```

Comment:
The problem is that the english one is IMHO as well wrong since that message is the same for a freestyle projects